### ISSUE TITLE
RECMD -q option

### ISSUE THREAD

**Issue by randomaccess3** [EricZimmerman/Issues#124]
-q option to suppress processing details
particularly for searching would be good to only see whether there was a hit (and where) in a hive or not
Otherwise hard to follow when using -d

**Comment by EricZimmerman** [comment]
this is a tough one because the hive's processing messages are in the Registry parser side. the warnings, etc are important for people to see, but this should help

https://github.com/EricZimmerman/Issues/issues/161
